cmd/kgo-repeater/main.go

```diff
@@ -83,9 +103,12 @@ func main() {
 	}
 
 	log.Info("Getting topic metadata...")
-	opts := []kgo.Opt{
-		kgo.SeedBrokers(strings.Split(*brokers, ",")...),
-	}
+	wConfig := worker.NewWorkerConfig(
+		"kgo", *brokers, *trace, *topic, *linger, *maxBufferedRecords, *useTransactions, *compressionType, *compressiblePayload, *username, *password, *enableTls)
+	opts := wConfig.MakeKgoOpts()
+	opts = append(opts, []kgo.Opt{
+		kgo.ProducerBatchMaxBytes(1024 * 1024),
```

Comment:
This batch size setting seems out of place with the rest of the change?

Reply:
Yeah agreed not the most elegant...  I needed to set `ProducerBatchMaxBytes` after adding these additional options to get the `client` (which is only used to get the topic meta data).

Modeled after what happens in `repeater_worker.go`:

https://github.com/redpanda-data/kgo-verifier/blob/4eeb1bc08dec52c14654144925db733847b57ee2/pkg/worker/repeater/repeater_worker.go#L267-L276